src/objects/DataImport__c.object

```diff
@@ -1441,7 +1441,7 @@
         <fullName>Payment_Declined_Reason__c</fullName>
```

Comment:
@callen-sfdo - Shouldn't the API name get updated the match the label too?

Reply:
@sam-knox Yeah as mike mentioned, we generally do not change API names unless its something that has not been released yet.  Basically changing the API name is the same as creating a new field and deprecating an old one, and any referencing code would need to be moved over.  It wouldn't necessarily be a big deal with just our code, but clients may be relying on it at this point.